Domain/Domain/Sources/UseCase/ManageWhiteboardObjectUseCase.swift

```diff
@@ -6,47 +6,117 @@
 //
 
 import Combine
+import Foundation
 
 public final class ManageWhiteboardObjectUseCase: ManageWhiteboardObjectUseCaseInterface {
     public var addedObjectPublisher: AnyPublisher<WhiteboardObject, Never>
     public var updatedObjectPublisher: AnyPublisher<WhiteboardObject, Never>
     public var removedObjectPublisher: AnyPublisher<WhiteboardObject, Never>
+
     private var whiteboardObjects: [WhiteboardObject]
 
     private let addedWhiteboardSubject: PassthroughSubject<WhiteboardObject, Never>
     private let updatedWhiteboardSubject: PassthroughSubject<WhiteboardObject, Never>
     private let removedWhiteboardSubject: PassthroughSubject<WhiteboardObject, Never>
+    private let selectedObjectIDSubject: CurrentValueSubject<UUID?, Never>
+    private let whiteboardRepository: WhiteboardObjectRepositoryInterface
```

Comment:
whiteboardObjectRepositoryInterface 타입이라 whiteboardObjectRepository가 구현체로 들어올 것 같습니다!!
```suggestion
    private let whiteboardObjectRepository: WhiteboardObjectRepositoryInterface
```
위와 같이 스타일을 변경하는 것이 어떨까요?

Reply:
아주 좋은 생각입니다 꼼꼼딴~~